ui/src/components/AppLayout/AppLayout.tsx

```diff
@@ -1,25 +1,13 @@
 import React from 'react'
 import { Layout, Spin } from 'antd'
-import { View } from '@tesler-ui/core'
-import AppSider from '../AppSider/AppSider'
+import AppSide from '../AppSider/AppSide'
 import AppBar from '../AppBar/AppBar'
 import { useDispatch, useSelector } from 'react-redux'
 import { AppState } from '../../interfaces/storeSlices'
-import { Card } from '../Card/Card'
 import DevPanel from '../DevPanel/DevPanel'
 import { SSO_AUTH } from '../../actions/types'
 import styles from './AppLayout.module.css'
-import {CustomFieldTypes} from "../../interfaces/widget";
-import MultipleSelectField from "../../fields/MultipleSelectField/MultipleSelectField";
-
-const skipWidgetTypes = [
-    'HeaderWidget',
-    'SecondLevelMenu'
-]
-
-const customFields = {
-    [CustomFieldTypes.MultipleSelect] : MultipleSelectField
-}
+import WidgetsView from '../WidgetsView/WidgetsView'
```

Comment:
It's better to operate in simple Tesler primitives in demo - screen, view, widget. There is nothing on `<WidgetsView />` in documentation.

Reply:
@Dergash renamed